classes/class3/homework/routes/index.js

```diff
@@ -0,0 +1,74 @@
+var mongoose = require('mongoose');
+
+var mongoURI = process.env.MONGOURI || "mongodb://localhost/test";
+mongoose.connect(mongoURI);
+
+var catSchema = mongoose.Schema({
+	name: String,
+	age: Number,
+	colors: [String]
+});
+
+var Cat = mongoose.model('Cat',catSchema);
```

Comment:
Ideally, schemas and models would be defined in a separate file in a `models` folder.